### PR TITLE
`many_glyphs` `recompute-text` option

### PR DESCRIPTION
**Pull request by ickshonpe**
# Objective

Add a commandline argument to the `many_glyphs` that forces the recomputation of all the text every frame.

### CODE REVIEW

File: examples/stress_tests/many_glyphs.rs [hunk 1]
```diff
@@ -10,18 +13,23 @@ use bevy::{
...
-        .add_systems(Startup, setup)
-        .run();
+        }),
+        ..default()
+    }))
+    .add_plugin(FrameTimeDiagnosticsPlugin::default())
+    .add_plugin(LogDiagnosticsPlugin::default())
+    .add_systems(Startup, setup);
+
+    if std::env::args().any(|arg| arg == "recompute-text") {
```
Comment (TimJentzsch) [inline examples/stress_tests/many_glyphs.rs:28]: Personally I think a flag like `--recompute-text` makes more sense here.

**Review by TimJentzsch — COMMENTED** [review]
I'm not quite sure how well this fits an example, I would think that they are meant to demonstrate some functionality to the user.
This looks more like a benchmark or a test of the internal implementation.

**Review by cart — APPROVED** [review]

File: examples/stress_tests/many_glyphs.rs (hunk 1 above)
Comment (cart) [reply examples/stress_tests/many_glyphs.rs:28]: Agreed

Comment (mockersf) [reply examples/stress_tests/many_glyphs.rs:28]: most examples currently just use the name of the configuration, not as a flag:

https://github.com/bevyengine/bevy/blob/main/examples/stress_tests/many_cubes.rs#L54
https://github.com/bevyengine/bevy/blob/main/examples/stress_tests/many_buttons.rs#L38
https://github.com/bevyengine/bevy/blob/main/examples/stress_tests/many_lights.rs#L88

**Review by mockersf — APPROVED** [review]

File: examples/stress_tests/many_glyphs.rs (hunk 1 above)
Comment (cart) [reply examples/stress_tests/many_glyphs.rs:28]: Cool cool. Lets stay consistent for now.

**Comment by cart** [comment]
> I'm not quite sure how well this fits an example, I would think that they are meant to demonstrate some functionality to the user.

This is a good reflex, but for stress tests I'm cool with being less strict about this because they exist to test performance more than they exist to be illustrative examples.

**Pull request merged**